### ISSUE TITLE
title of toolbox features in english or custom titles

### ISSUE THREAD

**Issue by tlorusso** [JohnCoene/echarts4r#3]
is it possible to switch language of the toolbox-elements as the default appears to be chinese? Or even name them by being able to pass some sort of title="" argument in the e_toolbox() function?  By the way : thanks for your great package.

**Comment by JohnCoene** [comment]
Reinstall the package.

```r
devtools::install_github("JohnCoene/ehcarts4r")
```

You can now pass additional options to `...`, see [official documentation](https://ecomfe.github.io/echarts-doc/public/en/option.html#toolbox.feature) for a full list.

```r
USArrests %>% 
  e_charts(UrbanPop) %>% 
  e_line(Assault) %>% 
  e_area(Murder, y.index = 1, x.index = 1) %>% 
  e_toolbox(feature = "saveAsImage", title = "Save As Image")
```

Note `e_datazoom` is a convenience function so if you want to rename the title for that you will have to go first initialise the toolbox with the `dataZoom` feature, then use `e_datazoom` (same for `e_brush`):

```r
USArrests %>% 
  e_charts(UrbanPop) %>% 
  e_line(Assault) %>% 
  e_area(Murder, y.index = 1, x.index = 1) %>% 
  e_y_axis(gridIndex = 1) %>%
  e_x_axis(gridIndex = 1) %>% 
  e_grid(height = "35%") %>% 
  e_grid(height = "35%", top = "50%") %>% 
  e_toolbox("dataZoom", title = list(zoom = "zoom", back = "back")) 
```

**Comment by tlorusso** [comment]
Thanks a lot! I get no error and it accepts the custom title. However the title gets cut ("Save As Im...") as the toolbox element is placed next to the container / canvas border. I've tried adjusting the position of the entire toolbox by passing the `right='20%'` as well as moving the position / allignment of the text with the `textPosition` / `textAlign` option. Is this supposed to work? 

I found a way that solved the problem for me in my specific case by putting a dataview element next to it. However if there is / can be a way to move the toolbox or the text OR the there would be more space for the title by default, that would be even better.
```
USArrests %>% 
  e_charts(UrbanPop) %>% 
  e_line(Assault) %>% 
  e_area(Murder, y.index = 1, x.index = 1) %>% 
  e_toolbox(feature = "saveAsImage", title = "Save As Image") %>% 
  e_toolbox(feature = "dataView",title="Data",lang="")
```
